src/initiate/initiate.ts

```diff
@@ -7,6 +8,12 @@ export class Initiate {
       slug: "initiate/token-swap",
       isDisabled: false,
       description: "Safely exchange tokens between multiple parties with customized vesting schedules.",
+      specialContent: `
```

Comment:
can we please use a slot so we don't have to put html and text copy in the viewmodel?

Reply:
this is actually another argument to ditch the javascript-generated boxes entirely.